sample_files/pre-commit-13.0/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Add the python requirements here.
+# Note: the libs in this file will be considered as "third-party libs" by isort.
```

Comment:
Not sure if this diff is good. It's going to introduce many conflicts probably, and as long as automations are transparent to the user, like the one you're proposing here, further explanations don't help much.

If you're gonna introduce instructions here, then IMHO it's better to just put a link, like "See http://whatever", leading to the corresponding section in our contributors guideline, where full explanations can be written.

Reply:
So should I only put an empty `requirements.txt` file?